Postfix_To_Infix/Postfix_To_Infix.c

```diff
@@ -0,0 +1,60 @@
+# include <stdio.h>
+# include <stdlib.h>
+# include <string.h>
+
+char stack[20];
+int top = -1;
+
+char pop()
+{
+    return stack[top--];//Pop the top value from the stack.
+}
+
+void push(char ch)
+{
+    stack[++top] = ch;//Push it onto the stack.
+}
+
+int operand(char x)//Checking if it is operator or letter
```

Comment:
Better to have return type as `bool` if you are returning a boolean value.

Also, have the comments about the function, above its definition for better readability. Try following this with other general comments too.

Reply:
@amaaniqbal I have done the required changes.Please review